channels/my.m3u

```diff
@@ -39,16 +42,46 @@ http://51.210.199.33/hls/stream.m3u8
 http://50.7.161.82:8278/streams/d/Naura/playlist.m3u8
 #EXTINF:-1 tvg-id="NTV7.my" tvg-name="NTV 7" tvg-country="MY" tvg-language="" tvg-logo="https://www.xtra.com.my/wp-content/themes/xtra_revamp/assets/img/ntv7.png" group-title="",NTV 7
 http://50.7.161.82:8278/streams/d/Rtmntv7/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="MY" tvg-language="" tvg-logo="https://myklikstatic.secureswiftcontent.com/image/TVlogotvokay.jpg" group-title="",OKEY (1080p)
+#EXTVLCOPT:http-referrer=https://rtmklik.rtm.gov.my
+https://rtmlive03.secureswiftcontent.com/rtmchannel/03_1080.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="MY" tvg-language="" tvg-logo="https://myklikstatic.secureswiftcontent.com/image/tokyo_ch1.png" group-title="Sport",RTM Tokyo 2020 Channel 1 (1080p)
+#EXTVLCOPT:http-referrer=https://rtmklik.rtm.gov.my
+https://rtm-olympic01.secureswiftcontent.com/rtm/olympic01_1080.m3u8
```

Comment:
tried all these link with user agent https://rtmklik.rtm.gov.my but not working

Reply:
hi   what  player   you are using ?